lib/modules/platform/gitea/index.ts

```diff
@@ -277,6 +278,13 @@ const platform: Platform = {
       throw new Error(REPOSITORY_EMPTY);
     }
 
+    if (repo.has_pull_requests === false) {
+      logger.debug(
+        'Repo does not have pull requests - throwing error to abort renovation',
```

Comment:
```suggestion
        'Repo does not have pull requests - aborting repository',
```
This is IMO unnecessary information for users.

Reply:
we probaly should also update the other messages. they use similar message